File: 4 - Estrutura de Repetição/Exercicio.15.py
#15. Faça um algoritmo para imprimir a quantidade de números
# ímpares entre um intervalo especificado pelo usuário. 

quantidadeNumeros = 0

numero1= int(input('Digite o primeiro valor: '))
numero2= int(input('Digite o segundo valor: '))

if numero2 <= numero1:
 print('Intervalo inválido.')
else: 
  for n in range(numero1,numero2 +1):
   if n %2 == 1: 
    quantidadeNumeros+=1 
  print(quantidadeNumeros)    

File: Livro - Módulo Java/4 - Estrutura de Repetição/Exercicio15.java
import java.util.Scanner;

public class Exercicio15 {
    public static void main(String[] args) {
        // Faça um algoritmo para imprimir a quantidade de números ímpares entre um intervalo especificado pelo usuário

        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o número inferior: ");
        int nInferior = scanner.nextInt();

        System.out.print("Digite o número superior: ");
        int nSuperior = scanner.nextInt();

        int qtdNum = 0;

        if (nSuperior <= nInferior) {
            System.out.println("Intervalo inválido");
        } else {
            for (int i = nInferior; i <= nSuperior; i++) {
                if (i % 2 != 0) {
                    qtdNum++;
                }
            }
            System.out.println(qtdNum);
        }
    }
}